### PR TITLE
Daria/name threads 2

### LINKED ISSUE #10: Naming threads
Idea from @benabraham:

> když v past vedle pasti nebo objevu někdo dá na začátek do [] tak by se to mohlo dostat do titulku
> [algoritmy] asůdkůlk ad aůsdksůadk ➡️ Past na Honza Javorek: algoritmy
> a vynechat ten den, protože to už bude specifický dost
> jakože regulár `^\[.+?\]`

Do we want this? Seems like a neat idea 🤔 This could be also a nice opportunity to have some tests added to the repository. I can help with setting them up.

### PR DESCRIPTION
**Pull request by dariagrudzien**
Name threads in `#můj-dnešní-objev` should follow the format `Past na {user}: {bla bla}`

Hopefully resolves #10

### CODE REVIEW

File: juniorguru_chick/bot.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ async def on_message(message: discord.Message) -> None:
     elif channel_name == "past-vedle-pasti":
         await create_thread(message, "{weekday} past na {author}")
     elif channel_name == "můj-dnešní-objev":
-        await create_thread(message, "{weekday} objev od {author}")
+        await create_thread(message, "{weekday} objev od {author}", "Past na {author}: {name}")
```
Comment (honzajavorek) [inline juniorguru_chick/bot.py:55]:
```suggestion
        await create_thread(message, "{weekday} objev od {author}", "Objev od {author}: {name}")
```

File: juniorguru_chick/bot.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ async def on_message(message: discord.Message) -> None:
     elif channel_name == "past-vedle-pasti":
         await create_thread(message, "{weekday} past na {author}")
```
Comment (honzajavorek) [inline juniorguru_chick/bot.py:53]:
```suggestion
        await create_thread(message, "{weekday} past na {author}", "Past na {author}: {name}")
```

**Review by honzajavorek — DISMISSED** [review]
Mixing pasti and objevy 😄 Otherwise good. If you accept my suggestions, the code will be all right, but the tests will fail, so you'll have to fix that too.

**Comment by dariagrudzien** [comment]
Thanks for the catch!

**Review by honzajavorek — APPROVED** [review]

**Comment by honzajavorek** [comment]
Awesome!

**Pull request merged**